doc/source/notebooks/theory/cglb.pct.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+# %%
+import numpy as np
+import matplotlib.pyplot as plt
+from cglb import load_snelson_data, plot_prediction
+from pathlib import Path
+from gpflow.models import CGLB, SGPR, GPR
+from gpflow.kernels import SquaredExponential
+from gpflow.optimizers import Scipy
+
+# %%
+# %matplotlib inline
+
+# %% [markdown]
+# First, we load the `snelson1d` training dataset.
+
+# %%
+data = load_snelson_data()
+
+# %% [markdown]
+# The CGLB model introduces less bias in comparison to SGPR model.
+# We can show empirically that CGLB has a lower bias by plotting the objective landscape with respect to different values of the lengthscale hyperparameters.
+
+# %%
+
+x, y = data
+n = x.shape[0]
+m = 10
+
+iv_indices = np.random.choice(range(n), size=m, replace=False)
+iv = x[iv_indices, :]
+
+noise = 0.1
+gpr = GPR(data, kernel=SquaredExponential(), noise_variance=noise)
+cglb = CGLB(data, kernel=SquaredExponential(), noise_variance=noise, inducing_variable=iv)
+sgpr = SGPR(data, kernel=SquaredExponential(), noise_variance=noise, inducing_variable=iv)
+
+
+def loss_with_changed_parameter(model, parameter, value: float):
+    original = parameter.numpy()
+    parameter.assign(value)
+    loss = model.training_loss()
+    parameter.assign(original)
+    return loss.numpy()
+
+
+ls = np.linspace(0.01, 3, 100)
+losses_fn = lambda m: [loss_with_changed_parameter(m, m.kernel.lengthscales, l) for l in ls]
+gpr_obj = losses_fn(gpr)
+sgpr_obj = losses_fn(sgpr)
+cglb_obj = losses_fn(cglb)
+
+plt.plot(ls, gpr_obj, label="GPR")
+plt.plot(ls, sgpr_obj, label="SGPR")
+plt.plot(ls, cglb_obj, label="CGLB")
+plt.legend()
+plt.show()
+
+
+# %% [markdown]
+# The interface of CGLB model is similar to SGPR, however, it has extra options related to the conjugate gradient which is involved in the computation of `yᵀK⁻¹y`.
```

Comment:
Consider using latex math mode as opposed to quotes for math since this will be converted to markdown.